card/mod/05_standard/set/type/set.rb

```diff
@@ -1,6 +1,6 @@
 
 format :html do
-
+    
```

Comment:
Let's try to stay clean on invisible spaces.  I know that TextMate is bad about introducing this stuff.